src/nvim/ops.c

```diff
@@ -5704,18 +5705,23 @@ size_t op_register_amount(void)
 
 /// Set register to a given value
 ///
-/// @param[in]  name  Register name.
-/// @param[in]  reg   Register value.
```

Comment:
Better save the previous line and make this line have one *less* space: aligning parameter name descriptions makes diffs larger and requires more work (I mean, to reformat if needed), so I no longer do this. AFAIK most other contributors do neither.